setup.cfg

```diff
@@ -65,7 +65,7 @@ all =
 
```

Comment:
In the setup file you should declare the options for pre-commit and its hooks (see phys2bids setup.cfg)